### PR TITLE
Alert recording status in IRC

### PR DESCRIPTION
**Pull request by clehner**
- Detect recording start/stop events from Jitsi, and bridge them into IRC.
- Only announce recording URLs in chat if a recording was started at some point.

### CODE REVIEW

File: lib/manage.js [hunk 1]
```diff
@@ -94,15 +95,17 @@ export async function manage({meeting, meetings, xmppOptions}) {
           ircSay(info);
           xmppSay({info, meeting, xmppClient});
 
-          const oggUrl = logUrl.replace('-irc.log', '.ogg');
-          info = `Raw audio at ${oggUrl}`;
-          ircSay(info);
-          xmppSay({info, meeting, xmppClient});
+          if (recorded) {
```
Comment (msporny) [inline lib/manage.js]:
```suggestion
          if(recorded) {
```

File: lib/manage.js [hunk 2]
```diff
@@ -158,6 +161,18 @@ export async function manage({meeting, meetings, xmppOptions}) {
...
         }
       }
+
+      // recording stopped or started
+      if(stanza.is('presence') &&
+        stanza.getChild('jibri-recording-status')) {
+        const recStatus = stanza.getChild('jibri-recording-status').attrs.status;
+        if(recStatus === 'on') {
+          recorded = true;
+          ircSay('Recording is on.');
```
Comment (msporny) [inline lib/manage.js]:
```suggestion
          ircSay('This meeting is now being recorded.');
```

File: lib/manage.js [hunk 3]
```diff
@@ -158,6 +161,18 @@ export async function manage({meeting, meetings, xmppOptions}) {
...
+        } else if(recStatus === 'off') {
+          ircSay('Recording has stopped.');
```
Comment (msporny) [inline lib/manage.js]:
```suggestion
          ircSay('The recording for this meeting has ended.');
```

**Review by msporny — CHANGES_REQUESTED** [review]
Minor changes requested, otherwise LGTM. Thanks a ton for contributing to the cgbot, @clehner! Very much appreciated!

File: lib/manage.js (hunk 3 above)
Comment (clehner) [reply lib/manage.js]: Committed: 396d80faf6fb99a14e92cb18deecefc37ba36103

File: lib/manage.js (hunk 2 above)
Comment (clehner) [reply lib/manage.js]: Committed: 396d80faf6fb99a14e92cb18deecefc37ba36103

File: lib/manage.js (hunk 1 above)
Comment (clehner) [reply lib/manage.js]: Committed: 887809caae299790beef6ac58b4e0a05e321da0d

**Comment by clehner** [comment]
@msporny you're welcome; changes applied.

**Pull request merged**